src/modules/addTask.js

```diff
@@ -0,0 +1,23 @@
+// Importing the Task class from './task.js'
```

Comment:
- [ ] 🙏Please remove the commented code👩‍💻 in your project. This will help make your code cleaner🧹 and more readable.

Reply:
Dear Sir, I have implemented the changes as per your kind feedback and I am really thankful to you for that.